huaweicloud/services/workspace/resource_huaweicloud_workspace_service.go

```diff
@@ -95,6 +95,49 @@ func securityGroupSchemaResource() *schema.Resource {
 	}
 }
 
+func AssistAuthSchemaResource() *schema.Resource {
```

Comment:
please use the lowercase if we do not export this function.
```suggestion
func assistAuthSchemaResource() *schema.Resource {
```

Reply:
fixed